t/api/10-jobgroups.t

```diff
@@ -479,4 +479,91 @@ subtest 'prevent create/update duplicate job group on same parent group' => sub(
     );
 };
 
+subtest 'update default_keep_logs_in_days and default_keep_results_in_days' => sub() {
+    # TODO: refactor this test by extracting the common code from the next test
+    # parent group test
+    $t->post_ok(
+        '/api/v1/parent_groups',
+        form => {
+            name => 'test_parent_group',
+            default_keep_logs_in_days => 888,
+            default_keep_results_in_days => 777,
+        }
+    )->status_is(400, 'if default_keep_logs_in_days greater than default_keep_results_in_days create should fail')
```

Comment:
I guess if you solve your TODO in https://github.com/os-autoinst/openQA/pull/6522/files#diff-c1af4e5c4c843b331bde9bcfcb93d638a6fe8bc375927237d286c24b2cf5e128R318 by extracting helper constants/functions then also here you don't need to redo complete (heavy) requests over and over again but simply check function calls which should make this code easier as well